GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -308,6 +308,7 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(RevisionGridControl.Commands.GoToParent, Keys.Control | Keys.P),
                     Hk(RevisionGridControl.Commands.GoToChild, Keys.Control | Keys.N),
                     Hk(RevisionGridControl.Commands.NextQuickSearch, Keys.Alt | Keys.Down),
+                    Hk(RevisionGridControl.Commands.OpenCommitsWithDifftool, Keys.None),
```

Comment:
This applies generally unless the tab `Diff` has the focus.
That's why I'd rather add it to FormBrowse. Furthermore, it has concurring hotkeys "Open...".

Reply:
I still prefer RevisionGrid, I believe this should be available in other grids
But I will change

Reply:
Kept in RevisionGrid, as adding a menu

Reply:
You are right.

Hotkeys -- or more generic actions -- for visible but not focused controls is another construction site. But as Igor wrote: "One step a time".

So for the time being, the best would be to add this Hotkey to RevisionGrid _and_ to FormBrowse.

Reply:
I would have preferred to only add it to RevisionGrid at this point, to not duplicate, but added